transcendence-app/src/swagger-main.ts

```diff
@@ -21,7 +21,7 @@ async function bootstrap() {
     .build();
   const document = SwaggerModule.createDocument(app, options);
   const yamlString: string = yaml.stringify(document, {});
-  writeFileSync('./swagger-spec.yaml', yamlString);
+  writeFileSync('./swagger-spec/swagger-spec.yaml', yamlString);
   app.close();
 }
 bootstrap();
```

Comment:
what does the .keep file do?

Reply:
We create the swagger spec files in this directory, and we add this empty file to add this directory to git.
Otherwise, we should create it in the swagger e2e tests or when generating the spec files.